config/sync/system.site.yml

```diff
@@ -4,7 +4,7 @@ slogan: ''
 page:
   403: ''
   404: ''
-  front: /node/
+  front: /node/7
```

Comment:
I think this should be kept as /node in git until we are sharing a database with the content

Reply:
Ah right no worries I'll change that back.